docs/source/api/apollo-server.mdx

```diff
@@ -581,6 +581,17 @@ The `executeOperation` method takes two arguments:
 - The first argument is an object describing the GraphQL operation to be executed.
   - Supported fields are listed in the table below.
 - The second argument is the optional options object. This object includes the operation's optional [`contextValue`](../data/context#the-contextvalue-object). This argument is only optional if your server _doesn't_ expect a context value (i.e., your server uses the default context because you didn't explicitly provide another one).
+  
+```js
+const response = await server.executeOperation({
+  query: 'query SayHelloWorld($name: String) { hello(name: $name) }',
+  variables: { name: 'world' },
+}, {
+  contextValue: async () => {
+    return { userId: 'test' };
+  }
```

Comment:
For `executeOperation` we expect that the `context` function has already been resolved, yielding the `contextValue` object.

```suggestion
  contextValue: { userId: 'test' },
```

Reply:
good catch!